cpp.properties

```diff
@@ -20,8 +20,10 @@ compiler.clang10.exe=/usr/bin/clang++-10
 compiler.clang10.name=clang 10
 compiler.clang11.exe=/usr/bin/clang++-11
 compiler.clang11.name=clang 11
+compiler.clang11.options=-std=c++1z -Wall -Wextra -Wshadow -O3 -m32 -march=i386
 compiler.clangdefault.exe=/usr/bin/clang++-11
 compiler.clangdefault.name=clang default (11)
+compiler.clangdefault.options=-std=c++1z -Wall -Wextra -Wshadow -O3 -m32 -march=i386
```

Comment:
by doing so, aren't we obliging everyone to use 32 bit output? Could we make it optional? Or something like:

```
compiler.clang11x86.exe=/usr/bin/clang++-11
compiler.clang11x86.name=clang 11 x86
compiler.clang11x86.options=-std=c++1z -Wall -Wextra -Wshadow -O3 -m32 -march=i386
```